mvrx/src/main/kotlin/com/airbnb/mvrx/MavericksExtensions.kt

```diff
@@ -246,6 +246,31 @@ fun <V : Any> args() = object : ReadOnlyProperty<Fragment, V> {
     }
 }
 
+/**
+ * Fragment argument delegate that makes it possible to set fragment args without
+ * creating a key for each one.
+ *
+ * To create nullable arguments, define a property in your fragment like:
+ *     `private val listingId by argsOrNull<MyArgs?>()`
```

Comment:
```suggestion
 *     `private val listingId: MyArgs? by argsOrNull()`
```
Can you update the doc above, too?